pkg/logging/logging.go

```diff
@@ -235,26 +263,28 @@ func setupSyslog(logOpts map[string]string, tag string, debug bool) {
 	logrus.AddHook(h)
 }
 
-// setupFormatter sets up the text formatting for logs output by logrus.
-func setupFormatter() logrus.Formatter {
-	fileFormat := new(logrus.TextFormatter)
-	fileFormat.DisableTimestamp = true
-	fileFormat.DisableColors = true
-	switch os.Getenv("INITSYSTEM") {
-	case "SYSTEMD":
-		fileFormat.FullTimestamp = true
-	default:
-		fileFormat.TimestampFormat = time.RFC3339
```

Comment:
What's the rational behind this removal?

Reply:
my former comment on this got removed, my bad. I assumed this statement was not actually used given that the `logrus.Formatter` is configured with an hardcoded value for disabling the timestamp 🤔 **DisableTimestamp = true**

Reply:
Well that explains stuff... and I thought it was because of the detection of `os.Getenv("INITSYSTEM")`. @borkmann you usually run cilium-agent standalone. Did you ever miss the lack of timestamps?

Reply:
Nope, not yet so far. ;)